tfjs-core/src/backends/webgl/backend_webgl.ts

```diff
@@ -1597,7 +1585,7 @@ export class MathBackendWebGL extends KernelBackend {
     const b = this.exp(a);
     const sumExp = this.sum(b, axes).reshape(expandedShape);
 
-    return this.realDivide(b, sumExp) as T;
+    return b.div(sumExp);
```

Comment:
Hi Ann, isn't it going back to the public API? I thought we want to avoid this as much as possible, see this paragraph in the modularization design doc: "When writing a kernel within a backend it is quite reasonable to call other kernels. We want to avoid using the public api to do this. Thus if functionality from another kernel is needed, that functionality should be factored into a separate file (with no side effects) and imported and used directly." Or am I missing something?

Reply:
Discussed offline, this is a intermediate step to avoid circular dependency. LGTM.